easybuild/framework/easyblock.py

```diff
@@ -154,11 +154,6 @@ def __init__(self, ec):
         self.skip = None
         self.module_extra_extensions = ''  # extra stuff for module file required by extensions
 
-        # modules interface with default MODULEPATH
-        self.modules_tool = modules_tool()
-        # module generator
-        self.module_generator = module_generator(self, fake=True)
-
         # modules footer/header
```

Comment:
maybe move this part to below the `modules_tool`? Not required but it's more of a logical order to me?